docker/openemr/flex-3.11/utilities/devtools.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/sh
+
+
+if [ $1 == "psr2-report" || $1 == "clean-sweap" ]; then
```

Comment:
Lets drop the clean-sweep check in just this conditional, since the psr2-fix below will basically do this already

Reply:
Also, surround all the $1 with quotes. for example:
```
if [ "$1" == "psr2-report" ]; then
```